src/sbt-test/sbt-scalafix/explicit-files/test

```diff
@@ -0,0 +1,13 @@
+> scalafixEnable
```

Comment:
This should not be needed since `DisableSyntax` is a syntactic rule. No need to change the test, just FYI